reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -1,35 +1,57 @@
 .heatmap {
   overflow-y: auto;
+  overflow-x: auto;
   max-height: 600px;
   height: 100%;
   background-color: #ffffff;
 }
 
-.heatmap table {
-  overflow-y: auto;
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
   top: 0;
   z-index: 2;
-  background-color: white;
+  background-color: inherit;
   color: #133a53;
 }
 
+.darkmode .table thead th {
+  border-bottom: none;
+  color: #f2f2f2;
+}
+
+.table td,
+.table th {
+  border-top: none;
+}
+
+.darkmode .heatmap {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+  border-radius: 10px;
+}
+
 .heatmap table th .subheading {
   font-weight: normal;
 }
 
 .heatmap table tr.area td {
   border: 0;
-  padding: 0;
   vertical-align: middle;
 }
 
-.heatmap table tr.area:first-child td {
-  padding-bottom: 10px;
+.darkmode .area {
```

Comment:
Put this below the .area block, and only override the border-top-color rather than border-top. Where do the color and background-color values come from in lightmode?

Reply:
✅

Reply:
Did you intend to remove the border-top-color altogether ?